python/gtsam/imuBias.py

```diff
@@ -0,0 +1 @@
+from .gtsam.imuBias import *
```

Comment:
Same comment, this trick should be documented :-)

Reply:
done